Common/Orders/Fees/BitfinexFeeModel.cs

```diff
@@ -67,9 +69,11 @@ public override OrderFee GetOrderFee(OrderFeeParameters parameters)
 
             unitPrice *= security.SymbolProperties.ContractMultiplier;
 
-            // apply fee factor, currently we do not model 30-day volume, so we use the first tier
+            // Determine fee and round to lot size decimal places
+            var decimalPlaces = security.SymbolProperties.LotSize.GetDecimalPlaces();
+            var fee = Math.Round(unitPrice * order.AbsoluteQuantity * feePercentage, decimalPlaces);
```

Comment:
Maybe something like `quantity -= quantity % security.SymbolProperties.LotSize;` might be simpler rounding logic? we use that approach in the rest of lean like: `BuyingPowerModel/OrderSizing`

Think we should review the live trading implementation at https://github.com/QuantConnect/Lean/blob/master/Brokerages/Bitfinex because I think the bug was originally reported for live trading and I don't think we use the feeModel directly? The fix here is still valid 👍